crates/router/Cargo.toml

```diff
@@ -93,7 +93,7 @@ rand = "0.8.5"
 rand_chacha = "0.3.1"
 rdkafka = "0.36.2"
 regex = "1.10.4"
-reqwest = { version = "0.11.27", features = ["json", "native-tls", "__rustls", "gzip", "multipart"] }
+reqwest = { version = "0.11.27", features = ["json", "__rustls", "rustls-tls", "gzip", "multipart"] }
```

Comment:
It should be sufficient to use `rustls-tls` alone, since that internally enables `__rustls`.